### ISSUE TITLE
Find an copyright-free movie and generate a set of thumbnails for every x seconds

### ISSUE THREAD

**Issue by flekschas** [flekschas/piling.js#93]
Another interesting example could be to combine [time curves](https://aviz.fr/~bbach/timecurves/) with visual piling. To do that we'd need to find a copyright-free movie, create thumbnails at every `x`th second (similar to what QuickTime does), and generate the movie's time curve.

**Comment by flekschas** [comment]
To compute the embedding curve ourself let's create a separate python script we're calling after we extracted the frames. The script should do the following: 

1. Load the frame images using https://docs.scipy.org/doc/scipy-0.13.0/reference/generated/scipy.ndimage.imread.html
2. Flatten the images and stack them together into a matrix such that a row represents an image
3. Run the images through [SKLearn's MDS](https://scikit-learn.org/stable/modules/generated/sklearn.manifold.MDS.html), [tSNE](https://scikit-learn.org/stable/modules/generated/sklearn.manifold.TSNE.html#sklearn.manifold.TSNE) and [UMAP](https://github.com/lmcinnes/umap)
4. Normalize the positions to `[0.1,0.9]` using https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html#sklearn.preprocessing.MinMaxScaler
5. Output a JSON file describing the items. Each item should be an object of the following form:
   ```javascript
   {
     src: './path/to/frame/image.png',
     mds: [u,v],
     tsne: [u,v],
     umap: [u,v]
   }
   ```
